### ISSUE TITLE
download_section_examples implemented?

### ISSUE THREAD

**Issue by fumitoh** [sphinx-gallery/sphinx-gallery#376]
I tried to remove download buttons on each gallery page by setting `download_section_examples` to `False` but it didn't work. I tried to look into the source but didn't find relevant parts. Is this setting in effect?

**Comment by Titan-C** [comment]
It is true. This option is not in the source code anymore. To be honest
I wasn't even aware this was a feature we support, not sure how it got
into the docs.

**Comment by fumitoh** [comment]
In turn, `download_all_examples` for summary pages is working, but doesn't seem to be documented!!

**Comment by Titan-C** [comment]
Yes, I was looking at the source and git history as well. It all comes back to my mind, I was me who implemented the feature. Initially the download button was per section, but after a discussion with users and contributors we settled for a global and unique download button containing all examples. PR #132 
The docs did not got updated after that change

**Comment by fumitoh** [comment]
Oh, okay I was probably confused in the first comment. What I was trying to do was to remove download buttons on **each example page**, but I guess no option was available to do that in the first place, correct?

**Comment by Titan-C** [comment]
No, you can't do that in the configuration. But you can hide the download buttons using CSS

**Comment by fumitoh** [comment]
Thanks for the tip!
